components/download.html

```diff
@@ -7,7 +7,7 @@
tn-thirdary more-btn">在线定制</a><p class="center">可自由选择所需图表和组件进行打包下载</p><h3>下载后……</h3><p><a href="./tutorial.html">5 分钟上手 ECharts</a></p></div><div class="d-section-gl d-section"><h2>下载 ECharts GL 1.1.1</h2><p>ECharts GL 是 WebGL 的扩展包，提供了丰富的三维可视化组件以及常规图表的性能增强。</p><h4>选择需要的版本 <span class="warn">注：开发环境建议选择源代码版本，该版本包含了常见的警告和错误提示。</span></h4><div class="list-wrap row"><div class="col-xs-6 col-sm-3 col-sm-offset-3"><a target="_blank" href="undefined/echarts-gl.min.js"><div class="circle-wrap"><span class="mode">压缩版</span><span class="size">637 KB</span></div></a></div><div class="col-xs-6 col-sm-3"><a target="_blank" href="undefined/echarts-gl.js"><div class="circle-wrap"><span class="mode">源代码</span><span class="size">1.74 MB</span></div></a></div></div><a href="https://echarts.baidu.com/option-gl.html" class="btn btn-thirdary btn-two">查看 GL 文档</a><a href="https://echarts.baidu.com/examples.html#chart-type-globe" class="btn btn-thirdary btn-two">查看 GL 实例</a></div></div><footer><div class="container"><div class="row"><div class="col-md-9"><p>Apache ECharts is an effort undergoing incubation at The Apache Software Foundation (ASF), sponsored by the Apache Incubator. Incubation is required of all newly accepted projects until a further review indicates that the infrastructure, communications, and decision making process have stabilized in a manner consistent with other successful ASF projects. While incubation status is not necessarily a reflection of the completeness or stability of the code, it does indicate that the project has yet to be fully endorsed by the ASF.</p><p>Copyright © 2017-2019, The Apache Software Foundation Apache ECharts, ECharts, Apache, the Apache feather, and the Apache ECharts project logo are either registered trademarks or trademarks of the Apache Software Foundation.</p></div><div id="footer-icon-panel" class="col-md-3"><a href="https://www.apache.org"><img src="images/asf_logo.svg" class="footer-apache-logo"></a><div class="icon-panel"><a href="mailto:dev@echarts.apache.com?Body=%28Thanks%20for%20using%20ECharts.%20Email%20us%20if%20you%20have%20non-technical%20problems%20using%20ECharts.%20For%20technical%20support%2C%20please%20go%20to%20https%3A//github.com/ecomfe/echarts/issues%20.%29" class="footer-icon"><img src="images/icon-email.png"></a><a href="https://twitter.com/EChartsJs" class="footer-icon"><img src="images/icon-twitter.png"></a><a href="https://weibo.com/u/5160877841" class="footer-icon"><img src="images/icon-weibo.png"></a><a href="https://github.com/ecomfe/echarts" class="footer-icon"><img src="images/icon-github.png"></a></div></div></div></div></footer></div><script type="text/javascript" src="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/js/bootstrap.min.js"></script><script type="text/javascript" src="./js/common.js"></script><script src="js/log.js"></script><script src="js/download.js?_v_=1572272992682"></script><script type="text/javascript">document.getElementById('nav-download').className = 'active';
```

Comment:
This file is generated when release a new version. So please don't include them in the PR.